frontend/src/components/TaskValueCreatedVisualization.tsx

```diff
@@ -57,11 +61,24 @@ export const TaskValueCreatedVisualization: FC<{
   }, [version.tasks, customers]);
 
   return (
-    <StakesVisualization
-      customerStakes={data}
-      width={width}
-      height={height}
-      vertical
-    />
+    <div className={classes(css.stakes)}>
+      <div style={{ width, height }}>
+        <StakesVisualization
```

Comment:
```suggestion
        <CustomerStakesVisualization
```

By "inlining" I meant this (and removing the now unused `StakesVisualization` component).